pkgs/top-level/all-packages.nix

```diff
@@ -19630,6 +19630,8 @@ with pkgs;
 
   spooles = callPackage ../development/libraries/science/math/spooles { };
 
+  spoolman-frontend = callPackage ../by-name/sp/spoolman/frontend.nix { };
```

Comment:
Wait, this is legal?

Reply:
If I remember correctly, I took that from other existing packages.